sormas-ui/src/main/java/de/symeda/sormas/ui/contact/ContactDataForm.java

```diff
@@ -101,7 +101,8 @@
                     fluidRowLocs(ContactDto.UUID, ContactDto.EXTERNAL_ID) +
                     fluidRowLocs(ContactDto.REPORTING_USER, ContactDto.REPORT_DATE_TIME) +
                     fluidRowLocs(ContactDto.REGION, ContactDto.DISTRICT) +
-                    fluidRowLocs(ContactDto.CASE_ID_EXTERNAL_SYSTEM, "") +
```

Comment:
Let's put the contact in one line with region and district instead. We are doing the same in the case in #1637